NOTICE

```diff
@@ -1,9 +1,2 @@
 OpenSearch
-Copyright 2021 OpenSearch Contributors
-
-This product includes software developed by
```

Comment:
Please confirm that asynchronous-search *does not* contain code from Elasticsearch or the Apache Software Foundation, and that it was just a bad copy-paste?

Reply:
We have dependencies on core and ASF however this plugin is solely developed from scratch by the Opendistro community members

Reply:
Dependencies != code copied, good